gulpfile.js

```diff
@@ -8,7 +8,27 @@
    gulp/tasks/default.js specifies the default set of tasks to run
    when you run `gulp`. */
 
+const gulp = require( 'gulp' );
 const requireDir = require( 'require-dir' );
 
 // Require all tasks in gulp/tasks, including subfolders.
 requireDir( './gulp/tasks', { recurse: true } );
+
+gulp.task( 'build',
+  gulp.series(
+    gulp.parallel(
+      'styles',
+      'scripts',
+      'images'
+    ),
+    'copy'
+  )
```

Comment:
The way you've made this a series makes sense to me intuitively, but I'm confused about why it didn't need to be that was before.

Reply:
before it was `'styles', 'scripts', 'images', 'copy'`, so in series for each task. This changes it so the first three run in parallel, which in theory should be more efficient.

Reply:
I thought that previously Gulp ran things in parallel unless told otherwise, hence the previous need for `runSequence` in some places?

Reply:
Ah right, you are correct. Thinking back to this, I think I had them all globbed together and it would fail when `copy` was included. We could try fiddling with this task and see if it can be simplified.

Reply:
Just to be clear: This makes sense as you have it here. I'm just not sure how copy didn't run before build stuff completed before :)